clients/lcdproc/main.c

```diff
@@ -36,8 +36,8 @@
 #include "mode.h"
 #include "shared/sockets.h"
 #include "shared/report.h"
-#include "shared/configfile.h"
-#include "getopt.h"		/* This is our local getopt.h! */
```

Comment:
Did you remove getopt.h in this PR?

Reply:
It is still there, but I think I will remove it. I don't think there is any reason to use the local (13 year old) version.

Reply:
Yes, please remove all these parts which are not needed. Reducing amount of code is one of the main reasons for this whole endeavor.